sim/druid/savage_roar.go

```diff
@@ -47,8 +48,8 @@ func (druid *Druid) registerSavageRoarSpell() {
 
 		ApplyEffects: func(sim *core.Simulation, _ *core.Unit, spell *core.Spell) {
 			druid.SpendComboPoints(sim, spell.ComboPointMetrics())
-			druid.TigersFuryAura.Duration = time.Duration(float64(time.Second) * (9.0 + (5.0)*float64(druid.ComboPoints())))
-			druid.TigersFuryAura.Activate(sim)
+			druid.SavageRoarAura.Duration = durationBonus + time.Duration(float64(time.Second)*(9.0+(5.0)*float64(druid.ComboPoints())))

```

Comment:
You should precompute the final duration number outside of the lambda for performance reasons (so its not calculated on every call)

Reply:
I just saw it depends on current combo points so maybe precompute as much as you can and leave only the combo points calculation in here.

Reply:
Also I'm noticing you're first spending combo points, then you're using `druid.ComboPoints()` wont thta always be 0 in this case?

Reply:
shoot, yea spend needs to go after, thanks